CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in the last 3 major versions of the postgresql cookbook.
 
+## v7.1.9 (2020-03-04)
```

Comment:
```suggestion
## Unreleased
```

Reply:
Please don't version bump in the changelog, we'll deal with this in a later PR. Thank you! :)